### PR TITLE
feat(setup): switch to official terramate-setup-action

### LINKED ISSUE #10: switch to official setup terramate action
https://github.com/terramate-io/terramate-action

### PR DESCRIPTION
**Pull request by kaiehrhardt**
https://github.com/terramate-io/terramate-action

fixes #10

### CODE REVIEW

**Comment by kaiehrhardt** [comment]
feels like rate limits. any idea @m4s-b3n

File: .github/workflows/test-and-release.yml [hunk 1]
```diff
@@ -57,9 +57,8 @@ jobs:
       - test-simple
     strategy:
       matrix:
-        go-version: ["1.17", "latest"]
-        terramate-version: ["v0.4.2", "latest"]
-        terraform-version: ["1.5.7", "latest"]
+        terramate-version: ["latest"]
+        terraform-version: ["latest"]
```
Comment (m4s-b3n) [inline .github/workflows/test-and-release.yml:61]: Test other versions than latest?

File: action.yml [hunk 2]
```diff
@@ -3,14 +3,14 @@ name: "Terramate All-In-One"
...
-    required: false
-    default: "latest"
   terramate-version:
     description: "The version of Terramate to use"
     required: false
     default: "latest"
+  terramate-cloud-organization:
+    required: false
+    description: "Terramate Cloud organization to use by default when using Terramate Cloud features"
+    default: ""
```
Comment (m4s-b3n) [inline action.yml:13]: Drop that

**Review by m4s-b3n — CHANGES_REQUESTED** [review]

File: .github/workflows/test-and-release.yml (hunk 1 above)
Comment (kaiehrhardt) [reply .github/workflows/test-and-release.yml:61]: which one? then we have to maintain that info... not sure...

Comment (m4s-b3n) [reply .github/workflows/test-and-release.yml:61]: before we had
```yaml
terramate-version: ["v0.4.2", "latest"]
terraform-version: ["1.5.7", "latest"]
```

**Comment by m4s-b3n** [comment]
> https://github.com/infinite-automations/terramate-all-in-one/actions/runs/9809251203/job/27086774760?pr=11
> 
> feels like rate limits. any idea @m4s-b3n

nope, as long as we can not say for sure 🙅

**Review by m4s-b3n — APPROVED** [review]

**Pull request merged**